### ISSUE TITLE
AddSwaggerGen() can't be found

### ISSUE THREAD

**Issue by darth30joker** [dotnet/AspNetCore.Docs#24125]
Hi there,

I followed the doc here, however, when I add below lines, VS shows error on `builder.Services.AddSwaggerGen();`:

```
builder.Services.AddEndpointsApiExplorer();
builder.Services.AddSwaggerGen();
```

I think I am missing some packages here. Can we list the packages we need to install in the doc please?

Thanks
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ab408035-6313-a183-9940-32de00e0e977
* Version Independent ID: ab408035-6313-a183-9940-32de00e0e977
* Content: [Tutorial: Create a minimal web API with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/min-web-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/min-web-api.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by darth30joker** [comment]
I see the issue here:

In the doc, the `Enable OpenAPI support` is not ticked, but to have the Swagger support, it has to be ticked.

![additional_information](https://user-images.githubusercontent.com/100747/144044569-27d0bbf0-fa54-4eeb-97eb-3984fdfd851b.png)

**Comment by Rick-Anderson** [comment]
That's how it works.

**Comment by darth30joker** [comment]
@Rick-Anderson  Hi Rick, the problem here is in the doc, this option is not ticked, and whoever follows this doc, may have the same issue like me. So I guess we should update the doc.

**Comment by serpent5** [comment]
Fixed in #24125.
